docker/run-qemu.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/expect
+set SSH_KEY [read [open "~/.ssh/id_rsa.pub" r]]
+set PORT1 [lindex $argv 0]
+set PORT2 [lindex $argv 1]
+set QEMU_MAC [lindex $argv 2]
+set FILENAME [lindex $argv 3]
+
+spawn qemu-system-x86_64 -device virtio-net-pci,netdev=net0,mac=$QEMU_MAC \
+-netdev user,id=net0,hostfwd=tcp::$PORT1-:22,hostfwd=tcp::$PORT2-:23 \
+-drive if=virtio,file=$FILENAME,format=raw -no-reboot -vga vmware \
+-device virtio-rng-pci -drive if=pflash,format=qcow2,file=ovmf.qcow2 \
+-vga std -cpu kvm64 -enable-kvm -m 256 -serial mon:stdio --nographic
+
+expect "root@intel-corei7-64:~#"
+# Wait for boot to finish completely
+sleep 5
```

Comment:
Hopefully this sleep was the thing that finally got this stable as previously most of the time the QEMU testing failed. I still want to test this in CI few times more to make sure. test this please